### ISSUE TITLE
Paste image from clipboard?

### ISSUE THREAD

**Issue by kylescousin** [champloohq/tusk#85]
Any way of pasting an image you have on the clipboard into a note? Eg. when you take a screenshot

**Comment by wjaskowski** [comment]
+1. I am also looking for that.

**Comment by beatwade** [comment]
I've tried many methods , find using shutter with key shortcut can be a good way, shutter can store screenshot to local and we can directly drag them to note to upload

**Comment by garrya** [comment]
I think this is the only feature missing right now :+1:

**Comment by Bryanx** [comment]
This feature would be really great!

**Comment by ediesyoum** [comment]
If this feature existed, I would use nothing but Tusk for my notes!

**Comment by mmati8** [comment]
I also be waiting for this features.

**Comment by chengwengao** [comment]
I'm also looking for this feature,it would be great to have this feature!

**Comment by aylinsgl** [comment]
This worked for me before but somehow stopped working a few weeks ago. Loved the feature I hope it returns!

**Comment by mmati8** [comment]
Very strange because sometimes it works and sometimes not. No repeatability

**Comment by MarcoR83** [comment]
yeah I also was under the impression that it worked for me with 0.23. Maybe Evernote changed something in their web-client that is not compatible with tusk anymore?

**Comment by Ir1d** [comment]
+1 on this, really looking forward

**Comment by HM102** [comment]
you can already do this in the new tusk version using the beta editor. You can also resize the image.

**Comment by jluismoran** [comment]
> you can already do this in the new tusk version using the beta editor. You can also resize the image.

@HM102 can you tell me how to test the beta editor? I just installed the latest version 0.23 and this feature is still not present

**Comment by matryosha** [comment]
To enable beta editor (and ability to paste an image from the clipboard) go to [Evernote Settings](https://www.evernote.com/Settings.action) and click "Go to new version" in Account Summary:
![image](https://user-images.githubusercontent.com/23679019/75632263-08c4e480-5c03-11ea-8964-cd630f4b0427.png)
After that refresh Tusk to get new editr

**Comment by frankydz** [comment]
> To enable beta editor (and ability to paste an image from the clipboard) go to [Evernote Settings](https://www.evernote.com/Settings.action) and click "Go to new version" in Account Summary:
> ![image](https://user-images.githubusercontent.com/23679019/75632263-08c4e480-5c03-11ea-8964-cd630f4b0427.png)
> After that refresh Tusk to get new editr

it work. Finally can paste pic from clipboard

**Comment by franciosi** [comment]
Thank you @matryosha. I can't open 'Evernote Settings' here, with Tusk 0.23.0 and Fedora 33. Does anyone have the same error or know why?

**Comment by matryosha** [comment]
@franciosi You don't have to switch to the new version inside Tusk. Login elsewhere using any browser and make that change there. As Tusk is not receiving any further development I suggest just start using Evernote using a browser of your choice. Also, something like [webapp-manager](https://github.com/linuxmint/webapp-manager) could make things handier.

**Comment by franciosi** [comment]
@matryosha , good tips, thank you
